full-moon/src/ast/mod.rs

```diff
@@ -39,7 +46,7 @@ mod versions;
 
 #[cfg(any(feature = "lua52", feature = "luajit"))]
 pub mod lua52;
-#[cfg(feature = "lua54")]
+#[cfg(any(feature = "lua54", feature = "cfxlua"))]
```

Comment:
Since `cfxlua` already extends lua54, we don't need to change this config since lua54 should always be enabled

```suggestion
#[cfg(feature = "lua54")]
```